readme.md

```diff
@@ -88,6 +89,9 @@
 - [Parcel — очень быстрый бандлер, не требующий настройки](https://habrahabr.ru/post/344486/) - Parcel quick review.
 - [Parcel — пишем плагин](https://habrahabr.ru/post/344858/) - How to write a Parcel plugin.
 
+### French
+
+- [Parcel - Présentation](https://www.grafikart.fr/tutoriels/javascript/parcel-bundler-985)
```

Comment:
Please provide a description.

` - [title](url) - description `

Reply:
My bad I fixed it